symplyphysics/laws/kinematic/vector/velocity_relative_to_reference_frame.py

```diff
@@ -57,9 +58,9 @@ def calculate_relative_velocity(
     time_ = symbols("time")
     position_ = scale_vector(
         time_ / time_change_,
-        add_cartesian_vectors(
+        subtract_cartesian_vectors(
```

Comment:
There are other places where subtract could be used. Do you want to find and update them as well?